src/SiteMaster/Core/Util.php

```diff
@@ -233,14 +233,23 @@ public static function getHTTPInfo($url, $options = array())
         
         curl_exec($curl);
 
-        $http_status = curl_getinfo($curl, CURLINFO_HTTP_CODE);
+        $http_status   = curl_getinfo($curl, CURLINFO_HTTP_CODE);
         $effective_url = curl_getinfo($curl, CURLINFO_EFFECTIVE_URL);
         $curl_error_no = curl_errno($curl);
 
         curl_close($curl);
         
+        //Always sleep after requesting data, this will help prevent loops from flooding a server.
+        sleep(1);
+        
         $okay = self::httpCodeIsOkay($http_status);
 
+        //Sometimes HEAD requests result in a false negative.  Do a GET request just to verify
+        if (true === $options[CURLOPT_NOBODY] && false === $okay) {
```

Comment:
Addressed in https://github.com/mfairchild365/site_master/commit/31402367ca5bdae637b14cdd8ccc574d4af3f378

This should be good for now.  However, I just thought of a good use for using the output.  The output could be hashed to determine if a page at one URL is likely the same page at a different URL.